arches/app/search/search_export.py

```diff
@@ -398,13 +398,14 @@ def to_geojson(self, instances, headers, name):  # a part of the code exists in
                             properties[header] = None
                 for key, value in instance.items():
                     if key == geometry_field:
-                        geometry = GEOSGeometry(value, srid=4326)
-                        for geom in geometry:
-                            feature = {}
-                            feature["geometry"] = JSONDeserializer().deserialize(GEOSGeometry(geom, srid=4326).json)
-                            feature["type"] = "Feature"
-                            feature["properties"] = properties
-                            features.append(feature)
+                        if isinstance(value, GeometryCollection):
```

Comment:
It might be better just to check if there is a value:
```
if value:
```
That should handle empty strings and an `None`. If value something other than one of those, then we probably want it to throw and error because that would help point to a different (and maybe more serious issue).

Reply:
@chiatt Part of the issue that that a string of 'None' is being passed as the value, which then makes the code fall over:

![image](https://github.com/archesproject/arches/assets/38533555/5a86dd09-db45-41eb-bc01-0162e06b3572)

That's why I went with the method of checking the value was a geometry